### ISSUE TITLE
3.1 Release - Test list

### ISSUE THREAD

**Issue by victorjonsson** [victorjonsson/Arlima#84]
The plugin have been in beta mode for over a year now. It's time to release a version that we can consider as "stable"

Here's a list of manual tests that has to be successfully gone through before release. Let us know if you have any suggestions!

- Can it be installed on an empty wp instance? (version 3.9 and 4.0)
- Can you install version 2.9 and then upgrade to version 3.1?
- Can a new list be created?
- Can the properties of a list be edited and saved.?
- Can a list be deleted?
- Can you search for wordpress posts and drag them over to a list? (does the list support child articles)?
- Can articles be dragged between lists?
- Can an article be copied and dragged?
- Can an article be deleted?
- Can you drag an article preset over to a list?

wip...

**Comment by chredd** [comment]
Suggestions: 

* Supported browsers: Chrome, Firefox, Safari, IE10+ (?)
* Does the widget work as expected?
* Is it possible to connect a list to page?
* Is the shortcode working as expected?

One thing that would be sweet for the first stable version is skipping Fancybox and fixing the mixed licenses so that we can get back into the wordpress.org plugin repo. Thoughts on that?

**Comment by victorjonsson** [comment]
I agree that it would be good for the project if Arlima was present in wordpress repo. The only thing is that it requires time/work that I/we dont have any to spare. 

If you're up to it your more than welcome to manage all that. I begun with creating a grunt task called "create-release". My idea was that the task should copy out the source cofe to a temp dir, then checkout the svn repo and move copied source into the repo and create a tag. Maybe theurs some easier way to do it :)

**Comment by chredd** [comment]
I can totally understand that. Bringing it back into the wp.org repo also gives us some kind of responsibility to take care of questions and what not from people using it - which also takes a lot of time.

Let me try to find some time and I'll take care of it. Your idea of a grunt task and SVN checkout sounds like a swell idea.
Let's revisit this question in a near future.
